exit_codes/exit_codes.py

```diff
@@ -58,3 +58,21 @@ class ExitCode(Base):
 
     # configuration error
     CONFIG = 78
+
+
+    # Aliases which have the same name as those in the standard
+    # library os module, without the leading EX_. These are to
+    # allow for easier porting as EX_FOO may be globally replaced
+    # with ExitCode.FOO.  For new code, prefer the easier to read
+    # names above.
+
+    DATAERR = 65
```

Comment:
I think these could refer to the earlier definitions, e.g. `NOINPUT = NO_INPUT`.

Reply:
A quick experiment (I'm in a hurry so maybe I'm being sloppy) shows that doing this also has the benefit of reusing the docstring from the original definition.

Reply:
I'm amazed that this works (which is why I didn't even try it), but you're right, it does, and it's definitely better.

In which namespace is the unqualified NO_INPUT on the right hand side being looked up? The class namespace doesn't exist at that point does it?

Reply:
I assume that this works because of `IntEnum`'s metaclass. I was surprised that you could give docstrings to individual enum values, but I guess it's taking care of that too. (I've breezed through it, and it's reasonably complex.)

Reply:
It works on regular classes too:

  >>> class Foo:
  ...     BAR = 42
  ...     SHEILA = BAR
  ... 
  >>> Foo.SHEILA
  42

Something interesting is happening here.

Reply:
Here's what I see when I capture the `locals` when `SHEILA` is defined:
```
In [1]: class Foo:
   ...:     BAR = 42
   ...:     SHEILA = dict(locals())
   ...:     

In [2]: Foo.SHEILA
Out[2]: {'BAR': 42, '__module__': '__main__', '__qualname__': 'Foo'}
```

So there's a namespace for the class definition. I need to go review this stuff a bit.

Reply:
I'm just surprised that the stuff in that namespace is available before the namespace has been populated when __prepare__ is invoked. It would be interesting to see the timing of __prepare__ w.r.t. assignments to these class attributes.

Normally class namespaces are not in scope, because they are not part of LEGB. I guess during class definition the class namespace *is* L.